documentation/pipelines/dcsazure_AzureSQL_to_AzureSQL_prof_pl.md

```diff
@@ -0,0 +1 @@
+../../dcsazure_AzureSQL_to_AzureSQL_prof_pl/README.md
```

Comment:
This is not very helpful. Better way would be to add hyperlink to the README file using the markdown link notation. I noticed that this is how other pipeline READMEs are written but we should change it to a link and not a relative file path which may be difficult for the user to follow (it was difficult for me to understand where the file is).

Reply:
I'll update directly the link to the file. I saw this from the existing thing and followed it.

Reply:
Updated to point directly to the readme

Reply:
Fixed the folder name 
![image](https://github.com/user-attachments/assets/af9ed034-b035-45e6-ab28-65b8f334147d)

Reply:
I think you posted it on wrong thread.

Reply:
haha. Weird.